packages/cxl-lumo-styles/scss/typography.scss

```diff
@@ -9,8 +9,8 @@ html {
    *
    * @see https://cdn.vaadin.com/vaadin-lumo-styles/1.5.0/demo/typography.html#font-family
    */
-  --lumo-font-family: "Roboto", sans-serif;
-  --cxl-lumo-font-secondary: "Libre Baskerville", "Georgia", serif;
+  --lumo-font-family: "lato", "Roboto", sans-serif;
+  --cxl-lumo-font-secondary: "lato", "Libre Baskerville", "Georgia", serif;
```

Comment:
Why fallback fonts are sefir while primary sans-serif?

Reply:
I'm not sure why this decision was made, they were there when I started the PR.

Reply:
I see my comment can be confusing ;)

```
--cxl-lumo-font-secondary: "lato", "Libre Baskerville", "Georgia", serif;
                             ^ is sans serif
                                                     ^ rest is serif
```

with update of "lato" I'd expect to make rest (fallback) font faces in the line sans-serif too

Reply:
I agree :) Fixed